js/main.js

```diff
@@ -0,0 +1,20 @@
+function handleClick(event)
+{
+  if (event.key === 'Enter' || event.target.className === "search-icon fas fa-search")
+  {
+    var movieName = document.querySelector('.title').value;
+    getMovieData(movieName.split(" ").join("+"));
+  }
+}
+document.querySelector('.search-icon').addEventListener('click',handleClick);
+document.querySelector('input').addEventListener('keydown',handleClick);
```

Comment:
Instead of doing this with both a `click` and `keydown` event handler, you can just change this to a `form` element with a single `submit` event handler